cms/teams/models.py

```diff
@@ -0,0 +1,37 @@
+from functools import cached_property
+from typing import ClassVar
+
+from django.db import models
+from wagtail.admin.utils import get_user_display_name
+from wagtail.search import index
+
+
+class Team(index.Indexed, models.Model):
+    identifier = models.CharField(max_length=255, unique=True)
```

Comment:
When management is allowed, this error as it is `unique` true and create view doesn't allow creation of this identifier. So we need to add that this the create view or disable uniqueness for now.